lib/ui/src/libs/key_events.js

```diff
@@ -2,14 +2,14 @@ import keycode from 'keycode';
 
 export const features = {
   FULLSCREEN: 1,
```

Comment:
Maybe it's a good time to also switch from numbers to literals here, what do you think?